hera_cal/lstbin.py

```diff
@@ -482,11 +482,8 @@ def lst_bin_arg_parser():
         "'2458042/zen.2458042.*.xx.HH.uv' '2458043/zen.2458043.*.xx.HH.uv' \n"
         "Consult lstbin.lst_bin_files() for further details on functionality.")
     a.add_argument('data_files', nargs='*', type=str, help="quotation-bounded, space-delimited, glob-parsable search strings to time-contiguous nightly data files")
-    a.add_argument("--lst_init", type=float, default=np.pi, help="starting point for universal LST grid")
```

Comment:
Why were these options removed? Just because these parameters are inferred from the files themselves now?

Reply:
ah it was removed b/c it that capability was actually deprecated in previous PRs and I just caught it lingering around when making this PR...